### ISSUE TITLE
rename tables

### ISSUE THREAD

**Issue by hxxperr** [hxxperr/CovidStuff#1]
это можешь поменять на 

df['date']=pd.to_datetime(df['date'])
df['day']=df['date'].apply(lambda x: x.day)
![image](https://user-images.githubusercontent.com/78909941/169948640-5508b06b-5bdd-4ad0-a244-a3ad54e82fcb.png)

или

df['date']=pd.to_datetime(df['date']) 
df['year']=df['date'].apply(lambda x: x.year)

там, возможно, некоторые уже установили значение stratify на df['location']. Поставьте на y как здесь
![image](https://user-images.githubusercontent.com/78909941/169948604-920edcfb-6678-494e-a227-73028f847cef.png)

bug ass heatmap

import pandas as pd 
import numpy as np 
from pprint import pprint 
pd.set_option('display.max_columns', None) 
 
import plotly 
plotly.offline.init_notebook_mode() 
 
from sklearn.datasets import load_boston 
data = load_boston() 
df = pd.concat([pd.DataFrame(data['data'], columns = data['feature_names']), 
 pd.DataFrame(data['target'], columns=['MEDV'])], axis=1)

import matplotlib.pyplot as plt 
from sklearn.feature_selection import SelectKBest 
from sklearn.feature_selection import chi2 
from sklearn.model_selection import train_test_split 
from sklearn.feature_selection import mutual_info_regression 
import plotly.express as px 
X = df.drop(['Danger'], axis=1) 
y = df['Danger'] 
 
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.30, random_state=42)

### 
# Code adopted from https://machinelearningmastery.com/feature-selection-for-regression-data/ 
### 
fs = SelectKBest(score_func=mutual_info_regression, k='all') 
# learn relationship from training data 
fs.fit(X_train, y_train) 
# transform train input data 
X_train_fs = fs.transform(X_train) 
# transform test input data 
X_test_fs = fs.transform(X_test) 
 
_dict_features={'Features':[], 'Score':[]} 
 
for score_feature, column in zip(fs.scores_, df.columns): 
 _dict_features['Features'].append(column) 
 _dict_features['Score'].append(score_feature) 
 
_df=pd.DataFrame(_dict_features) 
 
# plot the scores 
fig = px.bar(_df, x='Features', y='Score', title="Visualization of the dependence of the target variable on features") 
fig.update_layout(height=500, width=950) 
fig.show()
